### PR TITLE
Add "can you offer staff" question

### PR DESCRIPTION
**Pull request by brucebolt**
What
----

Adds two new pages:

- Can you offer staff
- What kind of staff can you offer?

These two pages have been modelled around the existing "Can you offer space" and "What kind of space can you offer" questions.

Screenshots
----

"Can you offer staff" question:
Screenshot 2020-05-08 at 13 44 48

"What kind of staff can you offer" question:
Screenshot 2020-05-10 at 14 34 16

Failure to select any type of staff:
Screenshot 2020-05-11 at 08 58 50

Failure to enter a number of staff:
Screenshot 2020-05-11 at 08 57 00

Failure to select how much you would charge:
Screenshot 2020-05-11 at 08 57 22

How to review
-------------

- Review code.
- Run application locally.  Go to `/offer-staff` and test behaviour of Yes/No options.  Go to `/offer-staff-type` and test various validation options (must select at least one staff type, number of staff required for each selected staff type and cost of staff must be selected).

Links
-----

Trello card: https://trello.com/c/MihuvL85

### CODE REVIEW

File: app/views/coronavirus_form/offer_staff_type.erb [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    hint: t("coronavirus_form.questions.offer_staff_type.offer_staff_description.hint"),
+    id: "offer_staff_type_offer_staff_description",
+    error_message: error_items("offer_staff_type_offer_staff_description"),
+    value: @form_responses[:offer_staff_description]
+  } %>
+
+  <%= render "govuk_publishing_components/components/radio", {
+    heading: t("coronavirus_form.questions.offer_staff_type.offer_staff_charge.title"),
+    name: "offer_staff_charge",
+    id: "offer_staff_type_offer_staff_charge",
```
Comment (alex-ju) [inline app/views/coronavirus_form/offer_staff_type.erb]: it seems like the `radio` component doesn't yet support `id` unfortunately; only an `id_prefix` used to generate distinct ids for each input. this means that the error summary doesn't point to anything at the moment, will raise a PR in `govuk_publishing_components` to address this, but not a blocker for this work to be released.

**Review by alex-ju — APPROVED** [review]
Nice one, Bruce. I've only found an issue that is not a blocker and shall be addressed separately, in the components gem.

**Comment by brucebolt** [comment]
I've branched this off https://github.com/alphagov/govuk-coronavirus-business-volunteer-form/pull/288, since that PR is likely to be merged soon.  Also refactored the view and check answers helper to make the new charge question consistent with those added in #288 (these changes are solely in the final commit).

File: app/views/coronavirus_form/offer_staff_type.erb (hunk 1 above)
Comment (brucebolt) [reply app/views/coronavirus_form/offer_staff_type.erb]: Thank you Alex.  I'll leave `id` here now, since there is no detrimental effect.

**Pull request merged**